addons/mail/static/src/components/follower_list_menu/follower_list_menu.xml

```diff
@@ -26,6 +26,7 @@
                                     class="o_FollowerMenu_follower dropdown-item"
                                     followerLocalId="follower.localId"
                                     t-on-click="_onClickFollower"
+                                    t-on-o-hide-follower-list-menu="_onClickFollower"
```

Comment:
We should avoid this if possible, but if it's really needed, each event should have its own method.

Reply:
@seb-odoo Done!